app/assets/stylesheets/components/_organizations.scss

```diff
@@ -0,0 +1,63 @@
+.organization-details {
+  h2, h3, h4, h5 {
+    margin: 0;
+    font: inherit;
+  }
+
+  h2, h4, h5 {
+    font-weight: 700;
+  }
+
+  h2{
+    font-size: 20pt;
+  }
+
+  body {
+    line-height: 1;
+  }
+
+  .border-above{
+    border-top:1px solid #bbb;
+    padding-top:8pt;
+    margin-bottom: 15pt;
+  }
+
+  .border-around{
+    border:1px solid #bbb;
+    padding:8pt;
+    margin-bottom: 15pt;
+  }
+
+  h3, h4, #primary-services{
+    font-size: 15pt
+  }
+
+  h5 {
+    margin-top: 8pt;
+  }
+
+  label{
+    color:#888;
+    font-style:italic;
+    font-weight:normal;
+  }
+
+  #col1, #col2 {
+    @include span-columns(4.5 of 9);
+  }
+
+  #col2 {
+    @include omega;
+  }
+
+  .chunked, .chunked li{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space